.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
not sure what this `idea` folder is, maybe should be added to `.gitignore` (the .gitignore under root, not this one)

Reply:
I have also added .idea to that git ignore as I think these files are generated by webstorm. Although it looks like they weren't removed. Will have another look